### PR TITLE
add deprecated directive

### PR DESCRIPTION
**Pull request by kindermax**

### CODE REVIEW

File: tests/test_federation/utils.py [hunk 1]
```diff
@@ -131,7 +132,12 @@ async def async_direct_link(ids):
     Node('CartItem', [
         Field('id', Integer, cart_item_resolver),
         Field('cart_id', Integer, cart_item_resolver),
-        Field('name', String, cart_item_resolver),
+        Field(
+            'name',
+            String,
+            cart_item_resolver,
+            directives=[Deprecated('do not use')]
```
Comment (vmagamedov) [inline tests/test_federation/utils.py:139]: `hiku.graph.Link` is also a "field" in GraphQL terms, so it also can have field-level directives

Comment (kindermax) [reply tests/test_federation/utils.py:139]: Thanks, will implement directives for links as well

Comment (kindermax) [reply tests/test_federation/utils.py:141]: done

**Comment by vmagamedov** [comment]
LGTM 👍

**Pull request merged**